### PR TITLE
implement --no-cache

### LINKED ISSUE #177: Request: Add --no-cache
I would to have a `--no-cache` option.

### PR DESCRIPTION
**Pull request by AkihiroSuda**
Close #177

Signed-off-by: Akihiro Suda <suda.akihiro@lab.ntt.co.jp>

### CODE REVIEW

File: build.go [hunk 1]
```diff
@@ -47,6 +47,7 @@ func (cmd *buildCommand) Register(fs *flag.FlagSet) {
 	fs.Var(&cmd.buildArgs, "build-arg", "Set build-time variables")
 	fs.Var(&cmd.labels, "label", "Set metadata for an image")
 	fs.BoolVar(&cmd.noConsole, "no-console", false, "Use non-console progress UI")
+	fs.BoolVar(&cmd.noCache, "no-cache", false, "Do not use cache when building the image")
```
Comment (jessfraz) [inline build.go]: line is 88 characters (lll)

Comment (AkihiroSuda) [reply build.go]: Can we fix that in a separate PR?
Other lines seem already exceeding the limit.

Comment (j3ssb0t) [inline build.go]: line is 88 characters (lll)

Comment (jessfraz) [reply build.go]: oh sorry this is a stupid bot

**Comment by jessfraz** [comment]
thanks!

**Pull request merged**